event/serialize_test.go

```diff
@@ -0,0 +1,33 @@
+package event
+
+import (
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/mesg-foundation/engine/protobuf/types"
+	"github.com/stretchr/testify/require"
+	"github.com/tendermint/tendermint/crypto"
+)
+
+var data = &Event{
+	Key:          "eventKey",
+	InstanceHash: sdk.AccAddress(crypto.AddressHash([]byte("10"))),
+	Data: &types.Struct{
+		Fields: map[string]*types.Value{
+			"foo": {
+				Kind: &types.Value_StringValue{StringValue: "bar"},
+			},
+		},
+	},
+}
+
+func TestHashSerialize(t *testing.T) {
+	require.Equal(t, "2:cosmos1ffzdc9fkggz2srlgp6grj32uc9sg9qvzu4jym8;3:eventKey;4:1:foo:3:bar;;;;", data.HashSerialize())
+	require.Equal(t, "cosmos186j7ddz0046caf63h566ez5nxd23svd44ga94h", sdk.AccAddress(crypto.AddressHash([]byte(data.HashSerialize()))).String())
```

Comment:
by default, the bech32 are using `cosmos` prefix.
we override it with:
```
cfg, err := config.New()
	if err != nil {
		panic(err)
	}
	cosmos.CustomizeConfig(cfg)
```
I would like to extract the prefix from the config and hardcode them somewhere so we could override the cosmos's prefix in an `init` function.